sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,146 @@
+
+    if (result == pdPASS) {
+        printf("Continuous measurement task started\n");
+        return true;
+    } else {
+        continuous_active_.store(false);
+        return false;
+    }
+}
+
+
+void SHTC3Sensor::stopContinuousMeasurement() {
```

Clang (14.0.6)

[warning] method 'stopContinuousMeasurement' can be made static

(readability-convert-member-functions-to-static)